ngraph/frontend/onnx/frontend/src/op/matmul_integer.cpp

```diff
@@ -0,0 +1,46 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "op/matmul_integer.hpp"
+
+#include <cstddef>
+#include <memory>
+#include <vector>
+
+#include "dequantize_linear.hpp"
+#include "matmul.hpp"
+#include "quantize_linear.hpp"
+
+namespace ngraph {
+namespace onnx_import {
+namespace op {
+namespace set_1 {
+OutputVector matmul_integer(const Node& node) {
+    const OutputVector& inputs = node.get_ng_inputs();
+
+    const auto& A = inputs.at(0);
+    const auto& B = inputs.at(1);
+    const auto& A_zero_point = (inputs.size() > 2) ? inputs.at(2) : ngraph::op::Constant::create(ngraph::element::i64, {1}, {0});
```

Comment:
The default zero points should match the A and B input types. You should change int64 to the element type of A and B respectively.
EDIT: but since they are converted to i32 afterwards, you can just create i32 constants by default.

Reply:
Done!